src/views/pages/students.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import StudentStore from '../../store/student-store';
+import Student from '../components/student.jsx';
+import StudentPageController from '../../controllers/pages/students';
+
+export default class Models extends ListenerComponent {
+
+    componentWillMount(){
+        StudentPageController.getStudents();
+    }
+
+    getStores() {
+        return [
+            StudentStore
+        ];
+    }
+
+    getState() {
+        return {
+            students: StudentStore.getAllStudents()
+        };
+    }
+
+    render() {
+        return (
+            <div id="students">
+                <h1>All Students</h1>
+                {this.state.students.sort(this.studentSort).map((student) => {
+                    return (
+                        <div key={student.id}>
+                            <Student student={student}  />
+                        </div>
+                    );
+                })}
+            </div>
+        );
+    }
+
+    studentSort(a, b){
+        let splitA, splitB, lastA, lastB;
+        splitA = a.name.split();
+        lastA = splitA[splitA.length - 1];
+        splitB = b.name.split();
+        lastB = splitB[splitB.length - 1];
+        return lastA.localeCompare(lastB);
+    }
```

Comment:
Yeah. I wasn't putting a lot of thought into it. And I'm not going to fight this but I'm pretty sure I read that String.prototype.split() was supposed to split on spaces if you didn't give it a separation character. Either way, I don't even think the sorting was a story requirement, and I was entirely aware of the messiness of going by name, and don't care to fix that because it's not a major concern and we have less than 10 weeks left.